pkgs/applications/video/gpu-screen-recorder/default.nix

```
@@ -1,14 +1,26 @@
-{ stdenv, lib, fetchgit, makeWrapper, pkg-config, cudatoolkit, glew, libX11
-, libXcomposite, glfw, libpulseaudio, ffmpeg }:
-
-stdenv.mkDerivation rec {
+{ stdenv
+, lib
+, fetchgit
+, makeWrapper
+, pkg-config
+, libXcomposite
+, libpulseaudio
+, ffmpeg
+, wayland
+, libdrm
+, libva
+, libglvnd
+, libXrandr
+}:
+
+stdenv.mkDerivation {
   pname = "gpu-screen-recorder";
-  version = "1.0.0";
+  version = "2023.11.18";
```

Is this an actual version number or just the commit date?

I'd recommend this in the latter case:

```suggestion
  version = "unstable-2023-11-18";
```

I first was confused that this might be some untagged release. Tools like nix-update use the same format when referencing commits from a branch.